packages/@okta/vuepress-site/docs/guides/authenticators-okta-email/main/aspnet/integratechallengemagiclink.md

```diff
@@ -355,24 +276,22 @@ if (idxContext != null)
 If the `otp` value is valid, the `AuthenticationStatus` property of the `AuthenticationResponse` object returned by `VerifyAuthenticatorAsync` is `Success`. In this case, call `AuthenticationHelper.GetIdentityFromTokenResponseAsync` to retrieve the OIDC claims information about the user and pass them into your application. The user has now signed in.
 
 ```csharp
-  switch (authnResponse.AuthenticationStatus)
-  {
-    // other case statements
+switch (authnResponse.AuthenticationStatus)
+{
+   // other case statements
 
-    case AuthenticationStatus.Success:
-      ClaimsIdentity identity = await AuthenticationHelper.
-        GetIdentityFromTokenResponseAsync(_idxClient.Configuration, authnResponse.TokenInfo);
-      _authenticationManager.SignIn(new AuthenticationProperties(), identity);
-      return RedirectToAction("Index", "Home");
-  }
+   case AuthenticationStatus.Success:
+   ClaimsIdentity identity = await AuthenticationHelper.
+      GetIdentityFromTokenResponseAsync(_idxClient.Configuration, authnResponse.TokenInfo);
+   _authenticationManager.SignIn(new AuthenticationProperties(), identity);
+   return RedirectToAction("Index", "Home");
 }
 ```
 
 If the `otp` value isn't valid or an `AuthenticationStatus` is returned that is not handled by your case statement, you should advise the user to return to the original tab in the browser and enter the `otp` value there to proceed.
 
 ```csharp
-    return View(new MagicLinkCallbackModel {
-        Message = $"Please enter the OTP '{otp}' in the original browser tab to finish the flow."
-    });
-}
+return View(new MagicLinkCallbackModel {
+   Message = $"Please enter the OTP '{otp}' in the original browser tab to finish the flow."
+});
 ```
```

Comment:
"Step 9. Complete challenge and sign user in" is missing from the content. Either remove it from the diagram or add it as a step in the content.